cleanup_pks.sh

```diff
@@ -2,5 +2,12 @@
 
 source `dirname "${BASH_SOURCE[0]}"`/util.sh
 
-# TODO uncomment if we use a PKS cluster per job
-# pks delete-cluster ${TS_G_ENV}-${CLUSTER_NAME} --non-interactive --wait
+fats_setup_gcloud
```

Comment:
gcloud is already setup, unless you're trying to pickup the envvar which I'm not convinced we need (see below)